<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EdgeDetectWidget</class>
 <widget class="QWidget" name="EdgeDetectWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>55</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EdgeDetectWidget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>511</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Edge Detect</string>
   </property>
   <widget class="QLabel" name="labelAlgorithm">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Algorithm</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditThreshold2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>40</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRun">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>60</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditThreshold1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelGradientThreshod">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Gradient Threshold</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelApertureSize">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Aperture Size</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbAlgorithm">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Canny</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEditApertureSize">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
